specifications/wallet/spec-light-mode.md

```diff
@@ -0,0 +1,123 @@
+# Specification: Light mode for cardano-wallet
+
+9th Feb 2022
+
+Status: DRAFT
+
+# Synopsis
+
+This document specifies a light-mode for `cardano-wallet`.
+This mode aims to make synchronisation to the blockchain faster by using a less trusted source for blockchain data.
+
+Light wallets often employ this strategy of using a less trusted data source; for example the [Nami][] wallet uses [Blockfrost][] as data source. The purpose of the light-mode of `cardano-wallet` is to make this "trust vs speed" trade-off readily available to downstream software such as [Daedalus][] with *minimal* changes to its downstream API.
+
+In addition, the "trust versus speed" trade-off will be partially obsoleted by [Mithril][] technology, which aims to give us "trust *and* speed" by providing verified ledger state data as opposed to block data. The act of implementing light-mode in `cardano-wallet` does not only offer immediate benefits, but will, in fact, partially prepare the codebase for Mithril technology.
+
+  [daedalus]: https://daedaluswallet.io
+  [nami]: https://namiwallet.io
+  [blockfrost]: http://blockfrost.io
+  [mithril]: https://iohk.io/en/research/library/papers/mithrilstake-based-threshold-multisignatures/
+
+# Motivation
+
```

Comment:
Perhaps something like these:

1. Allow users to operate their wallet without needing to wait for the node to download and validate the entire blockchain.
2. Allow users to run cardano-wallet and Daedalus on systems with 8GB of memory and <= 10GB of disk space.
3. Allow users to control where they sit on the trust vs convenience spectrum, depending upon their current circumstances.